Test/ArpTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using NUnit.Framework;
+using SharpPcap;
+using SharpPcap.LibPcap;
+
+namespace Test
+{
+    [TestFixture]
+    public class ArpTest
+    {
+        [Test]
+        public void TestArp([PcapDevices] DeviceFixture fixture)
+        {
+            var d = (LibPcapLiveDevice)fixture.GetDevice();
+            var arp = new ARP(d);
+
+            // timeout should not be null
+            Assert.IsNotNull(arp.Timeout);
+
+            // and we can set a timeout
+            arp.Timeout = new TimeSpan(0, 0, 2);
+
+            System.Net.NetworkInformation.PhysicalAddress localMAC = null;
+            foreach (var address in d.Addresses)
+            {
+                if (address.Addr.type == Sockaddr.AddressTypes.HARDWARE)
+                {
+                    localMAC = address.Addr.hardwareAddress;
+                    Console.WriteLine("{0} - {1}", d.Name, localMAC);
+                }
+            }
+
+            Console.WriteLine("{0} found localMAC of {1}", d.Name, localMAC);
+
+            if ((localMAC != null) && (localMAC.GetAddressBytes().Length != 0))
+            {
+                var destinationIP = new System.Net.IPAddress(new byte[] { 192, 168, 1, 1 });
+                try
+                {
+                    var physicalAddress = arp.Resolve(destinationIP);
```

Comment:
Are you sure `192.168.1.1` always available?

Reply:
I figured even if the arp timed out it would be ok.

Reply:
What do you think? Does it matter? should we try to use a known address?

Reply:
Try Google DNS: `8.8.8.8`

Reply:
That's a good idea, fixed.